slices/libunwind-13.yaml

```diff
@@ -1,4 +1,7 @@
 package: libunwind-13
+essential:
+  - libunwind-13_copyright
+
```

Comment:
nit: It would be nice to have a newline separating `package` and `essential`. (Same in few other files.)

Reply:
done